src/store/ChartStore.js

```diff
@@ -219,6 +218,8 @@ class ChartStore {
                     this.changeSymbol(this.state.symbol, this.state.granularity);
                 }
             });
+
+            this.onMarketClosedStatus();
```

Comment:
bring this down so that `onMarketClosedStatus` run after the code apply the active symbols, so the checking of closed status do not run again on first initial